roaring/roaring.go

```diff
@@ -3961,6 +3961,9 @@ func (op *op) UnmarshalBinary(data []byte) error {
 	_, _ = h.Write(data[0:9])
 
 	if op.typ > 1 {
+		if 1152921504606847000 < int(op.value) {
```

Comment:
This looks like a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)) - if we go with this as the fix, it would be great if this were instead a well-named constant, defined with the appropriate arithmetic, or at least a comment explaining where it comes from.